features/bookings.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Testing meeting room booking API
+  As a user
+  I want to book a meeting room
+
+  Background:
+    Given I am a registered user
+
+  Scenario: Booking an available meeting room
+    Given A meeting room is available
+    When I make a POST api request to endpoint '/v1/bookings' with below params:
+      | start_time | 2021-09-29 10:00:00 |
+      | end_time   | 2021-09-29 10:30:00 |
+      | date       | 2021-09-29 |
```

Comment:
Isn't a `user_id` also needed to make a booking as per - https://github.com/kirti09/meeting_management/pull/7/files#diff-a10ebf3e7fe762337f4414c99db5c712e7f8f6737649fb098bac3852a3364b19R36

Reply:
Initially there will no user and meeting room to pass the ids, we need to create both and pass the ids in payload. That is the reason I haven't added user_id and meeting_id in table.